pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -479,6 +479,12 @@ let
       for chromiumBinary in "$libExecPath/$packageName" "$libExecPath/libGLESv2.so"; do
         patchelf --set-rpath "${lib.makeLibraryPath [ libGL vulkan-loader pciutils ]}:$(patchelf --print-rpath "$chromiumBinary")" "$chromiumBinary"
       done
+
+      # replace bundled vulkan-loader
+      if [ -e "$libExecPath/libvulkan.so.1" ]; then
```

Comment:
Is there a reason why you check whether this file exists before applying your fix?

If this file ever changes, we won't be able to catch it just from the build.
And if there are e.g. electron-source versions where this file is missing, we should probably use version conditionals instead.

```suggestion
```

Reply:
I am probably being overly cautious, it is fine to remove the check.